docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,40 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop 3.3.0
+2021-04-08
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Bug fixes and minor changes
+
+- Fixed an issue related to docker-compose action. Fixes [docker/for-win#10564](https://github.com/docker/for-win/issues/10564).
+- Fixed launching Docker Desktop from the Dock. Fixes [docker/for-mac#5515](https://github.com/docker/for-mac/issues/5515).
+- Ensure `--add-host host.docker.internal:host-gateway` causes `host.docker.internal` resolves to the host IP, rather than the IP of the IP router. See [docker/for-linux#264](https://github.com/docker/for-linux/issues/264).
+- Prevent inter-container HTTP traffic being forwarded via the HTTP proxy to the host. Fixes [docker/for-mac#5476](https://github.com/docker/for-mac/issues/5476).
+- Prevent the deletion of files which have been added to the folder containing the VM disk image. Fixes [docker/for-mac#5486](https://github.com/docker/for-mac/issues/5486).
+- Fixed an issue where delta downloads caused an illegal instruction exception. Fixes [docker/for-mac#5459](https://github.com/docker/for-mac/issues/5459).
+- Apply domain-based HTTPS proxy `no_proxy` rules for encrypted connections by using TLS SNI. Fixes [docker/for-mac#2732](https://github.com/docker/for-mac/issues/2732).
+- Fixed an issue where the dashboard opened the container in browser with port 0.
+- Perform a filesystem check when starting the Linux VM.
+- Fixed pulling an image from Docker Hub from the dashboard.
+- Remove unused DNS name `docker.for.mac.http.internal`.
+- Detect Linux kernel crashes and escalate them to the user.
+
+### Upgrades
+
+- [Docker Compose 1.29.0](https://github.com/docker/compose/releases/tag/1.29.0)
+- [Compose CLI v1.0.12](https://github.com/docker/compose-cli/tree/v1.0.12)
+- [Linux kernel 5.10.25](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.76-83885d3b4cff391813f4262099b36a529bca2df8-amd64/images/sha256-0214b82436af70054e013ea51cb1fea72bd943d0d6245b6521f1ff09a505c40f?context=repo)
```

Comment:
Kernel link should be https://hub.docker.com/layers/docker/for-desktop-kernel/5.10.25-6594e668feec68f102a58011bb42bd5dc07a7a9b/images/sha256-80e22cd9c9e6a188a785d0e23b4cefae76595abe1e4a535449627c2794b10871?context=repo